frontend/src/static/css/v_summary.scss

```diff
@@ -0,0 +1,185 @@
+@import 'colors';
+
+/* summary chart */
+#summary,
+.summary-status {
+  text-align: center;
+}
+
+.summary-picker {
+  align-items: center;
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: center;
+  margin-bottom: 2rem;
+
+  &__section {
+    align-items: inherit;
+    display: flex;
+    flex: 0 1 auto;
+    flex-flow: inherit;
+    justify-content: inherit;
+  }
+
+  &__checkboxes {
+    label {
+      margin-left: .5rem;
+    }
+
+    span {
+      margin-left: .25rem;
+    }
+  }
+
+  .mui-textfield,
+  .mui-select {
+    margin: .5rem;
+    padding-right: 10px;
+  }
+
+  .mui-btn {
+    background: transparent;
+    box-shadow: none;
+    color: mui-color('grey');
+    font-size: .75rem;
+    font-weight: bold;
+    left: -8px;
+    margin: 0;
+    padding: 0;
+    vertical-align: middle;
+  }
+
+  .search_box {
+    align-items: center;
+    display: flex;
+  }
+
+  input {
+    font-size: .75rem;
+    padding-right: 10px;
+  }
+
+  label {
+    font-size: .75rem;
+    overflow-y: hidden;
+    text-align: left;
+    width: fit-content;
+  }
+
+  input,
+  select {
+    font-size: .75rem;
+  }
+}
+
+.summary-charts {
+  margin-bottom: 3rem;
+
+  &__title {
+    display: block;
+    font-weight: bold;
+    text-align: left;
+
+    &--index {
+      background: mui-color('black');
+      color: mui-color('white');
+      display: inline-block;
+      font-size: medium;
+      overflow: hidden;
+      padding: .1em .25em .1em .25em;
+      vertical-align: middle;
+    }
+
+    &--repo {
+      display: inline-block;
+      font-size: medium;
+      padding: .5rem;
+    }
+
+    &--contribution {
+      display: inline;
+    }
+  }
+
+  &__fileType--breakdown {
+    overflow-y: hidden;
+
+    &__legend {
+      display: inline;
+      float: left;
+    }
+  }
+}
+
+.summary-chart {
+  margin-bottom: 1rem;
+  overflow: auto;
+  position: relative;
+  text-align: left;
+
+  &__title {
+    clear: left;
+
+    & > * {
+      display: inline;
+      padding-right: .5rem;
+    }
+
+    &--index,
+    &--repo {
+      font-weight: bold;
+    }
+
+    &--index::after {
+      content: '.';
+      display: inline-block;
+    }
+
+    &--repo {
+      padding-right: .25rem;
+    }
+
+    &--button {
+      color: mui-color('grey');
+      cursor: pointer;
+      text-decoration: none;
+    }
+  }
+
+  &__ramp {
+    position: relative;
+
+    .overlay {
+      height: 100%;
+      position: absolute;
+      top: 0;
+
+      &.show {
+        background-color: rgba(mui-color('white'), .5);
+        border: 1px dashed mui-color('black');
+      }
+
+      &.edge {
+        border-right: 1px dashed mui-color('black');
+      }
+    }
+  }
+
+  &__contrib {
+    text-align: left;
+
+    &--bar {
+      background-color: mui-color('red');
+      height: .3rem;
+      margin-top: .1rem;
+    }
+
+    &--bar--fileType {
```

Comment:
Can you delete this block, please? It is deleted in a previous PR. Do CHECK that the block you delete and added back are exactly the same because there may be other merged PR touches the same file.

Reply:
Yup, deleted.